src/main/java/hudson/scm/subversion/WorkspaceUpdater.java

```diff
@@ -194,6 +194,11 @@ protected SVNRevision getRevision(ModuleLocation l) {
          * Returns {@link org.tmatesoft.svn.core.SVNDepth} by string value.
          *
          * @return {@link org.tmatesoft.svn.core.SVNDepth} value.
+         *
+         * @deprecated as of 2.10
+         *      Use SubversionSCM.ModuleLocation.getSVNDepthFor* functions to correctly interpret
+         *      module location depth options or SVNDepth.fromString directly if you've taken the
+         *      depth from another source.
          */
         protected static SVNDepth getSvnDepth(String name) {
```

Comment:
I would recommend to also add a `@Deprecated` annotation so that Jenkins tooling will be able to find deprecated usages